### PR TITLE
Fix some test flakiness

### PR DESCRIPTION
**Pull request by lbonn**
Some of the most recent failures: some tight timeouts and scripts racing for tcp ports.

In theory, `TestUtils::getFreePort` have the same problem but I'm not sure if it occurred in practice (also, could be annoying to fix in C++).

### CODE REVIEW

File: src/aktualizr_lite/test_lite.sh [hunk 1]
```diff
@@ -49,10 +48,16 @@ add_target foo2
...
-
-# curl http://localhost:$port/repo/image/targets.json | json_pp
+port=
+try=0
+while [[ -z $port  && $try -le 100 ]]; do
+    port=$(netstat -lpan 2> /dev/null | awk -F '[ :]+' "/$pid\\/python3/ {print \$5}")
+    sleep 0.2
+    try=$((try+1))
+done
+echo "http server listening on $port"
```
Comment (patrickvacek) [inline src/aktualizr_lite/test_lite.sh]: Would it be better to move this port-finding logic into a reusable script?

File: src/libaktualizr/primary/aktualizr_test.cc [hunk 2]
```diff
@@ -635,7 +635,7 @@ TEST(Aktualizr, FinalizationFailure) {
     EXPECT_EQ(current_version, SIZE_MAX);
 
     auto aktualizr_cycle_thread = aktualizr.RunForever();
-    auto aktualizr_cycle_thread_status = aktualizr_cycle_thread.wait_for(std::chrono::seconds(10));
+    auto aktualizr_cycle_thread_status = aktualizr_cycle_thread.wait_for(std::chrono::seconds(20));
```
Comment (patrickvacek) [inline src/libaktualizr/primary/aktualizr_test.cc]: :( Is there a way we can get rid of the explicit times? I guess 20 seconds isn't so bad, but still.

Comment (lbonn) [reply src/libaktualizr/primary/aktualizr_test.cc]: Hm I don't really know how to do that better. Waiting forever and relying on something else to timeout would probably lead to worse error messages.

Putting less load on CI machines is the best bet I would say and it's ongoing.

File: src/aktualizr_lite/test_lite.sh (hunk 1 above)
Comment (lbonn) [reply src/aktualizr_lite/test_lite.sh]: yes that's fair, I'll try something

File: src/libaktualizr/primary/aktualizr_test.cc (hunk 2 above)
Comment (patrickvacek) [reply src/libaktualizr/primary/aktualizr_test.cc]: No worries, it's fine. I just got triggered by the hardcoded numbers, you know, but they aren't always evil.

**Comment by lbonn** [comment]
Someone for another review after last changes? :)

**Pull request merged**